engine/src/main/java/org/terasology/rendering/nui/UIWidget.java

```diff
@@ -92,6 +92,12 @@
 
     void onBindEvent(BindButtonEvent event);
 
+    /**
+     * Returns preferred content size of this widget.
+     * @param canvas Canvas on which this widget is drawn.
```

Comment:
Add a blank line between the description and the param/returns list. Makes it look much better and reader-friendly. Do this everywhere